release_notes/addtl-release-notes.adoc

```diff
@@ -0,0 +1,90 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="addtl-release-notes"]
+= Additional release notes
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/servicebinding-document-attributes.adoc[]
+:context: addtl-release-notes
+
+Release notes for additional related components and products not included in the core xref:../release_notes/ocp-release-notes.adoc#ocp-release-notes[{product-title} {product-version} release notes] are available in the following documentation.
+
+[IMPORTANT]
+====
+The following release notes are for downstream Red Hat products only; upstream or community release notes for related products are not included.
+====
+
+[cols="4,7a",options="header"]
+|===
+|Component or product |Release note documentation
+
+|AWS Load Balancer (ALB) Operator
+|xref:../networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc#aws-load-balancer-operator-release-notes[AWS Load Balancer Operator release notes]
+
+|Compliance Operator
+|xref:../security/compliance_operator/compliance-operator-release-notes.adoc#compliance-operator-release-notes[Compliance Operator release notes]
+
+|{rh-dev-hub} Operator
```

Comment:
This table isn't in alphabetical order. Could we alphabetize it for easier reading?

Reply:
Discussed offline, but for posterity, mentioned in the PR description what the original idea was:

> Arranged alphabetically in the style of [docs.redhat.com/en/products](https://docs.redhat.com/en/products) (i.e., ignore the "Red Hat" or "Red Hat build of" prefix in the official product name, but respect the inclusion of "OpenShift").